lib/sidekiq/cron/job.rb

```diff
@@ -292,7 +292,7 @@ def initialize input_args = {}
         elsif @klass
           message_data = {
             "class" => @klass.to_s,
-            "args"  => @args,
+            "args"  => hide(@args, args["secret_args"] || [])
```

Comment:
Sorry, I did not dig to much into the code. So, messages are used not only for presentation, but also for processing, right? I should ponder on this PR more.